### PR TITLE
Add "wrap" to exports.

### PR DESCRIPTION
**Pull request by cahrens**
@stoeffel Here is my PR to fix the exports function.

### CODE REVIEW

File: tests/exports.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+var _ = require('underscore');
+var deepEqual = require('assert').deepEqual;
+var s = require('../dist/underscore.string');
```
Comment (stoeffel) [inline tests/exports.js]: could you change this to 
```diff
- var s = require('../dist/underscore.string');
+ var s = require('../');
```

**Comment by stoeffel** [comment]
btw. to run the tests locally you can just run `npm test` in your shell.

**Comment by cahrens** [comment]
@stoeffel I made the change you requested (also changed a usage of double quotes to single quotes, as it appears that you are standardizing on single quotes) and squashed the commit. Thanks for pushing out a release so quickly!

I took a look at #481 and #482, but don't really have the background to review them.

**Comment by stoeffel** [comment]
Thanks for the PR.

> I took a look at #481 and #482, but don't really have the background to review them.

No problem. Thanks anyway.

**Comment by stoeffel** [comment]
finally published `3.3.4`. had some troubles with the new build chain, but now it works fine. 
// cc @epeli

**Pull request merged**